flytekit/clis/flyte_cli/main.py

```diff
@@ -1675,6 +1677,76 @@ def update_launch_plan_meta(description, host, insecure, project, domain, name):
     _click.echo("Successfully updated launch plan")
 
 
+@_flyte_cli.command('update-cluster-resource-attributes', cls=_FlyteSubCommand)
+@_host_option
+@_insecure_option
+@_project_option
+@_domain_option
+@_optional_name_option
+@_click.argument('attributes', nargs=-1, type=_click.UNPROCESSED)
+def update_cluster_resource_attributes(host, insecure, project, domain, name, attributes):
+    """
+    Sets matchable cluster resource attributes for a project, domain and optionally, workflow name.
+
+    Use a -- to separate arguments to this cli, and attributes used for cluster resource configuration.
+    e.g.
+        $ flyte-cli -h localhost:30081 -p flyteexamples -d development update-cluster-resource-attributes -- cpu=1 \
+        memory=500M
+    """
+    _welcome_message()
+    client = _friendly_client.SynchronousFlyteClient(host, insecure=insecure)
+    cluster_resource_attributes = _ClusterResourceAttributes(_parse_args_into_dict(attributes))
+    matching_attributes = _MatchingAttributes(cluster_resource_attributes=cluster_resource_attributes)
+
+    if name is not None:
+        client.update_workflow_attributes(
+            project, domain, name, matching_attributes
+        )
+        _click.echo("Successfully updated cluster resource attributes for project: {}, domain: {}, and workflow: {}".
+                    format(project, domain, name))
+    else:
+        client.update_project_domain_attributes(
+            project, domain, matching_attributes
+        )
+        _click.echo("Successfully updated cluster resource attributes for project: {} and domain: {}".
+                    format(project, domain))
+
+
+@_flyte_cli.command('update-execution-queue-attributes', cls=_FlyteSubCommand)
+@_host_option
+@_insecure_option
+@_project_option
+@_domain_option
+@_optional_name_option
+@_click.argument('tags', nargs=-1, type=_click.UNPROCESSED)
+def update_execution_queue_attributes(host, insecure, project, domain, name, tags):
+    """
+    Tags used for assigning execution queues for tasks belonging to a project, domain and optionally, workflow name.
+
+    Use a -- to separate arguments to this cli, and attributes used for cluster resource configuration.
+    e.g.
+        $ flyte-cli -h localhost:30081 -p flyteexamples -d development update-cluster-resource-attributes -- critical \
```

Comment:
It seems this would be better with like a YAML?

Reply:
https://click.palletsprojects.com/en/7.x/options/#multi-value-options

Reply:
updated to use multi value options, thanks for the suggestions!